docs/content/guides/rows/row-sorting.md

```diff
@@ -1,79 +1,141 @@
 ---
 id: 6o0zftmc
-title: Row sorting
-metaTitle: Row sorting - JavaScript Data Grid | Handsontable
-description: Sort your data in ascending, descending, or a custom order, across one or multiple rows of the data grid.
+title: Rows sorting
+metaTitle: Rows sorting - JavaScript Data Grid | Handsontable
+description: Sort rows alphabetically or numerically, in ascending, descending or a custom order, across one or multiple columns.
 permalink: /row-sorting
 canonicalUrl: /row-sorting
+tags:
+  - row sorting
+  - column sorting
+  - columns sorting
+  - columnSorting
+  - multicolumn sorting
+  - multi-column sorting
+  - multiColumnSorting
 react:
   id: h4jfevxj
-  metaTitle: Row sorting - React Data Grid | Handsontable
+  metaTitle: Rows sorting - React Data Grid | Handsontable
 searchCategory: Guides
 ---
 
-# Row sorting
+# Rows sorting
 
-Sort your data in ascending, descending, or a custom order, across one or multiple rows of the grid.
+<p style="font-size:20px;">Sort rows alphabetically or numerically, in ascending, descending or a custom order, across one or multiple columns.</p>
 
 [[toc]]
 
-## Overview
-
-The [`ColumnSorting`](@/api/columnSorting.md) plugin works as a middleware between the data source and the Handsontable rendering module.
-
-The plugin can:
-* Map indices of displayed rows `visual indices` to the indices of corresponding rows in the datasource `physical indices` and vice versa
-* Alter the order of rows presented to a user without changing the datasource’s internal structure
-
-The sort operation is performed using a [stable sort algorithm](https://en.wikipedia.org/wiki/Sorting_algorithm#Stability) regardless of the browser you use or the size of the data set which you sort.
-
-::: tip
-Don't enable the [`ColumnSorting`](@/api/columnSorting.md) and [`MultiColumnSorting`](@/api/multiColumnSorting.md) plugins simultaneously.
-:::
-
-## Basic plugin configuration
-
-The simplest way to enable the plugin is to set the [`columnSorting`](@/api/options.md#columnSorting) option to `true`. You will then be able to use [the API methods](@/api/columnSorting.md#methods) and click on the header to sort, as shown in the example below:
-
-By default:
-
-* No column will be sorted initially
-* A sorting indicator will be enabled
-* Empty cells won't be sorted
-* The sort method will use default compare functions - read more about them [here](#default-compare-functions-sorting-different-kinds-of-data)
+Try it out: click on a column label to sort the rows in ascending, descending, or the original order.
```

Comment:
The more accessible approach, I think:
> In the demo showcase below presents the single-column sorting mode. Click on a column name to sort the data in a column in ascending, descending, or restore the original order.

Reply:
1. If possible, I'd avoid expresssions with "below" or "above":
    - If the layout changes, we need to change the content
    - It sounds stiff (formal register)
    - The MS style guide advises against it [as well](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/b/below)
2. "single-column sorting mode" – we're not explaining this term, we're not using it anywhere else.

How about this? [c72bac8](https://github.com/handsontable/handsontable/pull/10183/commits/c72bac8f8c65bacf61c825482506545723214003)